cmd/osbuild-worker-executor/handler_build.go

```diff
@@ -84,11 +84,12 @@ func runOsbuild(buildDir string, control *controlJSON, output io.Writer) (string
 	// #nosec G204
 	cmd = exec.Command(
 		"tar",
+		"-C",
+		buildDir,
```

Comment:
I'm curious why we serve the entire build dir, honest question there is nothing wrong with it (as such), but I wonder what the rational is. Does it make some internal things easier on the consumer side ?  

The thing that makes me slightly uncomfortable is that in some way anything that comes out of osbuild-worker-executor is potentially tainted/toxic so the less we touch it/process it the better. But again, not a blocker, mostly curiosity.

Reply:
the idea was to serve the build log through this as well, but in the end we just pull the output.

Reply:
oh right, because otherwise we're creating a tarball from the output directory, outputted to output/output.tar, and tar isn't always happy with creating an archive inside the directory you're archiving. So that's why it's one up. Either we server the entire build dir or we move the tarball after.